### ISSUE TITLE
pnp-react-search result webpart - "Something went wrong -can not set property 'PagingID' undefined" error

### ISSUE THREAD

**Issue by venkat061981** [microsoft-search/pnp-modern-search#665]
**Version used**
Ex: 3.7.0

**Describe the bug**
We have used **pnp-react-search-refiners** web part and customized as per clients requirement  to show search results from SharePoint online site. It was working fine but suddenly we came across with below error on chrome and edge browser. it's happening for internal users who are part of internal client network.Need experts help to resolve this issue. Appreciate your response.

**To Reproduce**
1. Go to 'https://abcd.sharepoint.com/sites/abcd/SitePages/Bulletins.aspx'
2. Click on 'Bulletin'
3. Scroll down to see the search results
![Search1](https://user-images.githubusercontent.com/30936026/104542763-2b5cc200-564a-11eb-85a4-40bafa7db01c.JPG)
![Search2-Error](https://user-images.githubusercontent.com/30936026/104542792-344d9380-564a-11eb-8fd4-b1c987dae8fa.JPG)

**Expected behavior**
Below result expected to happen.
![Search3-Expected Result](https://user-images.githubusercontent.com/30936026/104542856-4fb89e80-564a-11eb-84ba-5223bb2c0f7e.JPG)

**Screenshots**

**Desktop (please complete the following information):**

 - Browser [e.g. chrome, edge]

**Additional context**
NA

**Comment by wobba** [comment]
If you use v3.7 then update to latest and see if still an issue.

**Comment by venkat061981** [comment]
Thank you wobba for your response. Actually this issue has been resolved after some time and we have raised ticket with Microsoft. As per their investigation it's technical glitch due to some permissions issue but Microsoft didn't provide any root cause analysis.
Question: As per your suggestion if it's updated to latest version these type of issues won't occur again? Would it be possible to know root cause for these type of errors? appreciate your valuable feedback. Thanks
